src/generators/test/sampleTest.ts.hbs

```diff
@@ -1,32 +1,31 @@
-import { env, record, RecorderEnvironmentSetup, Recorder } from "@azure-tools/test-recorder";
+import { Recorder, RecorderStartOptions, env } from "@azure-tools/test-recorder";
 import * as assert from "assert";
+import { Context } from "mocha";
 
-const recorderEnvSetup: RecorderEnvironmentSetup = {
-  replaceableVariables: {
-    AZURE_CLIENT_ID: "azure_client_id",
-    AZURE_CLIENT_SECRET: "azure_client_secret",
-    AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
-    SUBSCRIPTION_ID: "azure_subscription_id"
-  },
-  customizationsOnRecordings: [
-    (recording: any): any =>
-      recording.replace(/"access_token":"[^"]*"/g, `"access_token":"access_token"`)
-  ],
-  queryParametersToSkip: []
+const replaceableVariables: { [k: string]: string } = {
```

Comment:
maybe we can use `Record<string, string>` instead of ` { [k: string]: string }`

Reply:
Sure, I will update it